services/middlewareServices/AuthMiddlewareService.js

```diff
@@ -47,20 +51,26 @@ class AuthMiddlewareService {
       const {
         access_token: retrievedToken,
         user_id: retrievedId,
+        isomer_user_id: isomerUserId,
       } = jwtUtils.verifyToken(isomercms)
+      if (!isomerUserId) {
+        const notLoggedInError = new Error("User not logged in with email")
+        notLoggedInError.name = "NotLoggedInError"
+        throw notLoggedInError
+      }
       const accessToken = jwtUtils.decryptToken(retrievedToken)
       const userId = retrievedId
       return { accessToken, userId }
     } catch (err) {
       logger.error("Authentication error")
-      if (err.name === "TokenExpiredError") {
-        throw new AuthError(`JWT token has expired`)
-      }
-      if (err.name === "JsonWebTokenError") {
+      if (err.name === "NotLoggedInError") {
         throw new AuthError(
           `Encountered an auth error at ${url}: ${err.message}`
         )
       }
+      if (err.name === "TokenExpiredError") {
+        throw new AuthError(`JWT token has expired`)
+      }
```

Comment:
I think this is because we wanted to maintain our errors as one for each basic error type (e.g. `NotFoundError` for 404 errors, `AuthError` for 401), with the appropriate information only in the messages. In this case I think we can just add in additional information for the thrown errors - what else should we be logging?

Reply:
we can just have a `NotLoggedInError` that extends the `AuthError` class then we can only care about the `TokenExpiredError` case in the `catch` blcok

Reply:
As discussed offline, for now we'll just throw an auth error with the appropriate message, 9a264f4